chains/celo/listener.go

```diff
@@ -33,3 +44,28 @@ func (l *listener) start() error {
 func (l *listener) close() {
 	l.conn.Close()
 }
+
+func (l *listener) getTransactionBlockHash(hash common.Hash) (blockHash common.Hash, err error) {
```

Comment:
I think it would be more clear to name this `getBlockHashFromTransactionHash` and `hash `-> `txHash`